test/suites/ACID.m

```diff
@@ -523,6 +523,7 @@
 function [stat] = logplot()
   stat.description = 'Test logscaled axes.';
   stat.unreliable = isMATLAB('>=', [8,4]); %FIXME: #590
+  stat.unreliable = stat.unreliable || isMATLAB('<=', [8,3]); %FIXME: #749 (Jenkins)
 
```

Comment:
Does that mean this is always unreliable?
[>=8,4] || [<=8,3] 
In any case i would directly put that in one line